### PR TITLE
jpa hw

### PR DESCRIPTION
**Pull request by Taniann**

### CODE REVIEW

File: src/main/java/ua/mateacademy/tania/lesson22/Office.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    @Column(name = "SALES", nullable = false, precision = 22, scale = 0)
+    public BigDecimal getSales() {
+        return this.sales;
+    }
+
+    public void setSales(BigDecimal sales) {
+        this.sales = sales;
+    }
+
+}
```
Comment (u2107680007911543774e7r) [inline src/main/java/ua/mateacademy/tania/lesson22/Office.java:76]: do you use any plugins to set column attributes?

**Review by u2107680007911543774e7r — APPROVED** [review]
gj

File: src/main/java/ua/mateacademy/tania/lesson22/Office.java (hunk 1 above)
Comment (Taniann) [reply src/main/java/ua/mateacademy/tania/lesson22/Office.java:76]: > do you use any plugins to set column attributes?

no

File: src/main/java/ua/mateacademy/tania/lesson22/OfficeDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        Locale.setDefault(Locale.ENGLISH);
+        factory = Persistence.createEntityManagerFactory("PERSISTENCE");
+    }
+
+    private static final Logger LOG = LogManager.getLogger(OrderDaoImpl.class);
+
+    private EntityManager entityManager = factory.createEntityManager();
+    @Override
+    public boolean insert(Office office) throws SQLException {
+        LOG.debug("persisting Orders instance");
```
Comment (u2107680007911543774e7r) [inline src/main/java/ua/mateacademy/tania/lesson22/OfficeDaoImpl.java:25]: Orders -> Offices ; )

File: src/main/java/ua/mateacademy/tania/lesson22/OfficeDao.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package ua.mateacademy.tania.lesson22;
+
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+public interface OfficeDao {
+    boolean insert(Office office) throws SQLException;
```
Comment (SerhiiPasko) [inline src/main/java/ua/mateacademy/tania/lesson22/OfficeDao.java]: Why do you throws SQLException?

Comment (SerhiiPasko) [reply src/main/java/ua/mateacademy/tania/lesson22/OfficeDao.java]: Clear all of those

**Review by SerhiiPasko — CHANGES_REQUESTED** [review]
Good. But clean unused throws

**Pull request merged**